athlinks-ontology.ttl

```diff
@@ -6,542 +6,692 @@
-
-#################################################################
-#    Annotation properties
-#################################################################
-
-###  http://purl.org/dc/elements/1.1/contributor
-dc:contributor rdf:type owl:AnnotationProperty .
-
-
-###  http://purl.org/dc/elements/1.1/creator
-dc:creator rdf:type owl:AnnotationProperty .
-
-
-###  http://purl.org/dc/elements/1.1/date
-dc:date rdf:type owl:AnnotationProperty .
-
-
-#################################################################
-#    Object Properties
-#################################################################
-
-###  http://adp.athlinks.com/schema/exerciseType
-:exerciseType rdf:type owl:ObjectProperty ;
-              rdfs:domain schema:ExerciseAction ;
-              rdfs:range :AthleticEventCategory ;
-              dc:creator "nextdude"^^xsd:string ;
-              dc:date "2017-07-26T14:00:17Z"^^xsd:dateTime .
-
-
-###  http://adp.athlinks.com/schema/legacyData
-:legacyData rdf:type owl:ObjectProperty ;
-            rdfs:domain owl:Thing ;
-            rdfs:range :LegacyDataRole ;
-            rdfs:comment "A legacy data record belonging to another system that is related to this object."@en ;
-            rdfs:label "legacy data"@en .
-
-
-#################################################################
-#    Data properties
-#################################################################
-
-
-### http://adp.athlinks.com/schema/age
-:age rdf:type owl:DatatypeProperty ;
-                 rdfs:domain schema:Person ;
-                 rdfs:range xsd:integer ;
-                 dc:creator "nextdude"^^xsd:string ;
-                 dc:date "2017-08-09T22:22:47Z"^^xsd:dateTime ;
-                 rdfs:comment "The age (in years) of a person."@en ;
-                 rdfs:label "Age"@en .
-
-###  http://adp.athlinks.com/schema/siblingOrdering
-:siblingOrdering rdf:type owl:DatatypeProperty ;
-                 rdfs:domain owl:Thing ;
-                 rdfs:range xsd:integer ;
-                 dc:creator "nextdude"^^xsd:string ;
-                 dc:date "2017-07-26T14:18:47Z"^^xsd:dateTime ;
-                 rdfs:comment "The numeric order of an object among siblings of the same property or type."@en ;
-                 rdfs:label "Sibling Ordering"@en .
-
-### TODO: should be remodeled
 :participatedIn a owl:ObjectProperty ;
-  rdfs:domain schema:Person ;
-  rdfs:range :ActivityType ;
-  rdfs:comment "An indication of a person participating in an ActivityType."@en ;
-  rdfs:label "participated in"@en .
+	rdfs:domain schema:Person ;
+	rdfs:comment "An indication of a person participating in an ActivityType."@en ;
+	rdfs:label "participated in"@en .
+# 
+# 
+# 
+# #################################################################
+# #
+# #    Data properties
+# #
+# #################################################################
+# 
+# 
+# http://adp.athlinks.com/schema/age
+
+:age a owl:DatatypeProperty ;
```

Comment:
Why do we need an age on a person? schema:Person already has DOB, right?  System-wise, keeping this up to date will be a pain when a simple function on the query or the result of the query will handle calculating the age. I understand that we store age with an entry but we aren't going to create a new person every time we create an entry, are we?

Reply:
I agree. `:age` makes sense on an entry-ish subject (e.g. `AthleticEventCompetitor`) but not on a `Person`.

Reply:
This is virtual. We compute it on the fly in the api, but we need the definition in place to make the schema stuff work out.

Reply:
AH! ok, as long as we don't keep it up to date in the DB I'm cool